ui/desktop/app/components/hidden-secret/index.hbs

```diff
@@ -4,26 +4,25 @@
 }}
 
 <div class='hidden-secret'>
-  <Rose::Button
-    @style='ghost'
-    @iconOnly={{if
-      this.isHidden
-      'flight-icons/svg/eye-16'
-      'flight-icons/svg/eye-off-16'
-    }}
+  <Hds::Button
+    @text={{t 'actions.toggle-secret'}}
+    @icon={{if this.isHidden 'eye' 'eye-off'}}
+    @isIconOnly={{true}}
+    @color='secondary'
     {{on 'click' this.toggleSecretVisibility}}
-  >
-    {{t 'actions.toggle-secret'}}
-  </Rose::Button>
-  <Copyable
-    @text={{this.displaySecretValue}}
-    @buttonText={{t 'actions.copy-to-clipboard'}}
-    @acknowledgeText={{t 'states.copied'}}
-  >
+  />
+
+  <div class='secret-content'>
     {{#if this.isHidden}}
       <pre>{{this.displaySecretValue}}</pre>
     {{else}}
       <span class='secret-mask'>{{this.textMask}}</span>
     {{/if}}
-  </Copyable>
+  </div>
+
+  <Hds::Copy::Button
```

Comment:
Should we prevent the height of the buttons from expanding as shown in this big secret?

<img width="589" alt="image" src="https://github.com/hashicorp/boundary-ui/assets/5783847/071870a7-6ed9-4ec2-9766-65e7295c2e60">

Reply:
@ZedLi looking into it

Reply:
Fix it! thanks for the catch!!